manifest.json

```diff
@@ -25,12 +25,28 @@
         }
       }
     },
-    "omnibox": {"keyword": "d"},
+    "omnibox": {"keyword": "ddg"},
     "options_page": "html/options.html",
     "background": {
             "scripts": ["js/background.js"]
     },
-    "permissions": [
-        "contextMenus"
-    ]
+    "chrome_settings_overrides": {
+      "search_provider": {
+		 "homepage": "http://www.duckduckgo.com",
+         "encoding": "UTF-8",
+         "favicon_url": "https://duckduckgo.com/favicon.ico",
+         "is_default": true,
+         "keyword": "duckduckgo.com",
+         "name": "DDG Search",
+         "search_url": "https://duckduckgo.com/?q={searchTerms}"
+      },
+	  "startup_pages": ["https://duckduckgo.com"]
+   },
+   "description": "Search the Web Privately with DuckDuckGo",
+   "icons": {
+      "128": "img/icon_128.png",
+      "16": "img/icon_16.png",
+      "48": "img/icon_48.png"
+   },
+	"permissions": [ "contextMenus", "unlimitedStorage", "http://*/*", "https://*/*", "tabs" ]
```

Comment:
@AdamSC1-ddg why are the permissions for `unlimitedStorage`, all websites and `tabs` needed? This will cause that  "Read and change data on all websites you visit" message in the install dialog.